.babelrc

```diff
@@ -1,6 +1,18 @@
 {
-  "presets": ["react", "es2015", "stage-0"],
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "browsers": ["last 2 versions", "ie >= 11"],
```

Comment:
this pr will never pass otherwise, or the minus repo would need to be moved to your account